lighthouse-core/audits/splash-screen.js

```diff
@@ -64,20 +65,20 @@ class SplashScreen extends MultiCheckAudit {
 
   /**
    * @param {LH.Artifacts} artifacts
+   * @param {LH.Audit.Context} context
    * @return {Promise<{failures: Array<string>, manifestValues: LH.Artifacts.ManifestValues}>}
    */
-  static audit_(artifacts) {
+  static async audit_(artifacts, context) {
     /** @type {Array<string>} */
     const failures = [];
 
-    return artifacts.requestManifestValues(artifacts.Manifest).then(manifestValues => {
-      SplashScreen.assessManifest(manifestValues, failures);
+    const manifestValues = await ManifestValues.request(context, artifacts.Manifest);
```

Comment:
I'll refrain from going promises->async/await in future changes to cut down the diffs as well :)